lib/cisco_node_utils/vxlan_vtep_vni.rb

```diff
@@ -195,10 +214,16 @@ def suppress_arp
     end
 
     def suppress_arp=(state)
-      # Host reachability must be enabled for this property
-      VxlanVtep.new(@name).host_reachability = 'evpn'
-      set_args_keys(state: (state ? '' : 'no'))
-      config_set('vxlan_vtep_vni', 'suppress_arp', @set_args)
+      if state
+        set_args_keys(state: '')
+        # Host reachability must be enabled for this property
+        VxlanVtep.new(@name).host_reachability = 'evpn'
+        config_set('vxlan_vtep_vni', 'suppress_arp', @set_args)
+      else
+        set_args_keys(state: 'no')
+        # Negate suppress-arp only if it is configured
```

Comment:
Please add a bit more context in the comments about why we only remove suppress-arp if it is configured.